src/commands/pack/tarballs.ts

```diff
@@ -9,6 +9,7 @@ Add a pretarball script to your package.json if you need to run any scripts befo
 
   static flags = {
     parallel: Flags.boolean({description: 'Build tarballs in parallel.'}),
+    'preserve-lockfiles': Flags.boolean({description: 'Preserve lockfiles in the tarball.'}),
```

Comment:
What's the argument for this being an opt-out flag? As opposed to an opt-in flag like `--remove-lockfiles`

Reply:
I think my preference is to make it an opt-in flag so that we don't risk suddenly breaking other people's builds - even if it's a small risk.

I have a list of of things to do in the next major. If we want, we can switch to an opt-out flag then